numpy/core/fromnumeric.py

```diff
@@ -3276,7 +3276,8 @@ def around(a, decimals=0, out=None):
     See Also
     --------
     ndarray.round : equivalent method
-
+    numpy.round: equivalent method
```

Comment:
```suggestion
```
`numpy.round` does not exist in the documentation so we cannot add it to the `See Also` for now. I think a good option would be to write in the `Notes` that `np.round` is often used as an alias for `np.around`. That would potentially clear up any confusion as to why `np.round` is used in the examples in the Notes section.